openedx/features/partners/migrations/0004_auto_20200320_0542.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.21 on 2020-03-20 09:42
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('partners', '0003_auto_20200120_0715'),
```

Comment:
Please update the dependency to `0004_partner_config_partneruser_status` as per [LP-1902](https://github.com/philanthropy-u/edx-platform/pull/1419/files)